python/src/gen3authz/client/arborist/base.py

```diff
@@ -1,28 +1,22 @@
-"""
-Define the ArboristClient class for interfacing with the arborist service for
-authz.
-"""
-
 import json
-import threading
+import sys
 from collections import deque
 
+import backoff
+import contextvars
+
+from ..base import AuthzClient
+
 try:
     import urllib.parse as urllib
 except ImportError:
     import urllib
-
-import backoff
+import httpx
 from cdislogging import get_logger
-import requests
 
-from gen3authz import string_types
-from gen3authz.client.arborist.errors import (
-    ArboristError,
-    ArboristUnhealthyError,
-    AuthZError,
-)
-from gen3authz.client.base import AuthzClient
+from ... import string_types
```

Comment:
relative imports scare me, but I think it's a personal problem, I just have seen that generally across the codebase the import paths are usually explicit. I don't have a super strong opinion though

Reply:
🤷‍♂️ https://stackoverflow.com/a/16748366/3316267